specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingSavingsPlan.json

```diff
@@ -0,0 +1,1204 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-04-01",
+    "title": "BillingManagementClient",
+    "description": "Billing client provides access to billing resources for Azure subscriptions."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/savingsPlanOrders/{savingsPlanOrderId}": {
+      "get": {
+        "description": "Get a savings plan order.",
+        "operationId": "BillingSavingsPlanOrder_Get",
+        "tags": [
+          "Savings plan order"
+        ],
+        "x-ms-examples": {
+          "SavingsPlanOrderGet": {
+            "$ref": "./examples/billingSavingsPlanOrderGet.json"
```

Comment:
The same feedback. Should this just be SavingsPlanOrderGet?

Reply:
Updated all examples file name to be savingsPlan...ByBillingAccount